src/components/AdminDashboard/CompleteIncident.js

```diff
@@ -247,7 +257,9 @@ const CompleteIncident = props => {
             {/* Tags */}
             <label className="label">
               Tags
-              <span style={{ fontWeight: "normal" }}>&nbsp;(comma separated values)</span>
+              <span style={{ fontWeight: 'normal' }}>
```

Comment:
I'm a little confused by what the previous code was doing here... specifically with (comma separated values) in a span?

Reply:
That was when I added the tags field to the completeincident form. It makes the label look like this:

**Tags** (comma separated values)

(Tags is bold and the rest is not)